### ISSUE TITLE
Leaving combat issues. 

### ISSUE THREAD

**Issue by Theguyxd** [Tercioo/Details-Damage-Meter#390]
Hey, ever since a few versions back I get a a frame hitch or stutter when leaving combat. Like, just when combat ends. I've narrowed it down to details. This never used to happen but it has been since 10.0. What could be the cause of this? It does not stutter/hitch upon entering combat or during combat. But as soon as combat is over and I leave combat I get a frame hitch. I'm at a loss.

**Comment by Theguyxd** [comment]
Seems to only manifest inside dungeons tho.

**Comment by Tercioo** [comment]
could you post a photo of this frame?

**Comment by Tercioo** [comment]
hmmm, okay, thanks for reporting, also make sure you don't have any 
memory counter showing in the screen as it may affect performance.

------ Original Message ------
From: "Theguyxd" ***@***.***>
To: "Tercioo/Details-Damage-Meter" 
***@***.***>
Cc: "Tercio Jose" ***@***.***>; "Comment" 
***@***.***>
Sent: 01-Dec-22 12:03:59 AM
Subject: Re: [Tercioo/Details-Damage-Meter] Leaving combat issues. 
(Issue #390)

**Comment by Theguyxd** [comment]
Thanks for taking the time to respond. I've looked around a bit and it does not seem to be a common issue for sure. But I'm at a loss here. I've reinstalled windows and set up the pc completely fresh with everything. DDU'd drivers, fully updated windows. Deploted everything to make sure dpc latency spikes are not occuring. Tested RAM and overclocks. I dont know what would be causing my framerate to tank exactly when leaving combat inside dungeons. What is actually happeing for this to be a thing? The only thing I can come up with is something to do with data being added to overall? I'm only running 1 window in details when this happens btw. Could it be an addon conflict? I don't think I'm running anyting else that would interfere with details in any way. 

All I have are some basic weakauras for my class, shadowedUnitFrames for myself and target and plater really.

Anyway, thanks for the time. It's super vague but maybe something comes to mind to what could be causing this.

**Comment by Theguyxd** [comment]
Small update: I updated to the latest alpha build and the stutter/hitch/small freeze didnt happen at the end of combat for about 3 dungeons. Then it started happening again. This leads me to believe something with regards to memory is happening maybe. But that is also strange since it has happend directly after starting the game before aswell.

**Comment by vellocets** [comment]
Wanted to chime in to say that this is happening to me and a few other people I've talked to as well. Happens inside and outside of dungeons/raids. Small frame hitch after combat drops. Disabling all addons stops it from happening, and re-enabling just Details causes it to start happening again.

**Comment by Tercioo** [comment]
Did you imported your profile from which person/streamers?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by vellocets** [comment]
> 

No, no imported profile for details. Just a normal install of the latest release version. Update interval is set to 1.5 and I have two windows, but I think that's the only thing I have changed.

**Comment by Flamanis** [comment]
Closing this issue as it is old and could have already been fixed. If you believe this to be in error, or if the issue still persists, please create a new issue.
